.github/pull_request_template.md

```diff
@@ -9,11 +9,16 @@
   - [ ] If a `manage.py` command is added, deleted, or modified its help text must
     be valid and it should be documented in detail in
     `docs/advanced/admin-commands.md`
-- Update `CHANGELOG.md`
+- Copy the changelog entry file template in `changeslogs/` to a new file in the
+  same directory following the `YYYY-MM-DD-name[-#].md` pattern with the day's
+  date, your preferred name, and an optional sequential number if you have
```

Comment:
Why "preferred name"?  I could see branch names or a PR number or something, but names could get confusing fast.  Whose name is put in a multi-user branch?  Stuff like that.  It's easy to sort out, but easier if we decide for them.

Obviously a PR number requires submitting a PR first (draft), so that is an extra burden, so I'd honestly suggest branch name, as those tend to be unique for a while, and almost certainly for a given day.

Reply:
Excellent point about multi-user branches. I like your idea of branch names better. I'll work on making that change :)